src/utility/robot_localization/params/ekf_template.yaml

```diff
@@ -36,7 +36,7 @@ debug: false
 debug_out_file: /path/to/debug/file.txt
 
 # Whether to broadcast the transformation over the /tf topic. Defaults to true if unspecified.
-publish_tf: true
+publish_tf: false
```

Comment:
is there something else being used in place of this?

Reply:
Nothing is using this tf at the moment.